core/compatibility/components/blocks/Category/Sidebar.js

```diff
@@ -1,4 +1,4 @@
-import { buildFilterProductsQuery } from '@vue-storefront/store/helpers'
+import { buildFilterProductsQuery } from '@vue-storefront/core/helpers'
```

Comment:
i think it should be in upgrade notes. Someone can use it, and its only simple unique string replacement :)

Or maybe better to leave old helpers with deprecation note and upgrade guide as a warning, and change them to invoke this new helpers? We could remove them in next releases more safely.